tendermint/src/rpc/endpoint/abci_info.rs

```diff
@@ -27,41 +27,58 @@ impl rpc::Response for Response {}
 
 /// ABCI information
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
 pub struct AbciInfo {
     /// Name of the application
     pub data: String,
 
     /// Version
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
 
     /// Last block height
-    pub last_block_height: block::Height,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub last_block_height: Option<block::Height>,
```

Comment:
This can only be empty on genesis, right? Maybe add a comment.

Reply:
Apparently that's how it's defined in Tendermint Go and v0.32 doesn't show it at all.

```
$ curl localhost:26657/abci_info
{
  "jsonrpc": "2.0",
  "id": "",
  "result": {
    "response": {
      "data": "{\"size\":0}",
      "version": "0.16.1",
      "app_version": "1"
    }
  }
}
```

Reply:
OK, looks like these fields where removed:
![image](https://user-images.githubusercontent.com/7441766/76803650-9d6e3b80-67da-11ea-895a-a3f7fc81ac1e.png)
![image](https://user-images.githubusercontent.com/7441766/76803686-aeb74800-67da-11ea-9056-741ec9d85983.png)
https://docs.tendermint.com/master/rpc/#/ABCI/abci_info

Reply:
Although the proto definition still contain these fields. Probably the best to keep it the way your dealt with it for now 👍

Reply:
Yeah. We'll have a few surprises like that with 0.33